tests/test_decoder.js

```diff
@@ -438,6 +467,74 @@ test('NbsDecoder.getPackets() returns the last packet of each type when given a
   });
 });
 
+test('NbsDecoder.getPacketByIndex() throws for invalid arguments', () => {
+  assert.throws(
+    () => {
+      decoder.getPacketByIndex();
+    },
+    /invalid type for argument `index`: expected integer/,
+    'NbsDecoder.getPacketByIndex throws for missing `index` argument'
```

Comment:
Same with the rest of these.

```suggestion
    'NbsDecoder.getPacketByIndex() throws for missing `index` argument'
```